modules/internal/ChapelLocale.chpl

```diff
@@ -917,6 +920,7 @@ module ChapelLocale {
   }
 
   pragma "no doc"
+  deprecated "locale.blockedTasks() is deprecated; please let us know if this is a problem for you"
```

Comment:
I don't love the "please let us know if this is a problem for you" phrasing, but I don't have any suggestions myself. If Brad is ok with this, that's fine by me.

Reply:
I don't think Brad is going to object to that. I adapted it from a [deprecation](https://github.com/chapel-lang/chapel/blob/6eb858a29adb580cdf6676cbc3b2e86ede4ba7f3/modules/internal/ChapelRange.chpl#L989) he added.

Reply:
I understand the hesitance, but didn't have a better idea.  In my case, I wanted it to be a "We don't think anybody's relying on this, but don't hesitate to contact us if you are" type of thing (rather than a "swear at us under your breath assuming nothing can change" type of thing), and I think that's the case here as well.  But I'm open to alternate phrasings as well if we come up with one we like better.

Reply:
I, in my guise as Politeness Man, actually rather liked it. Plus it'll only be there for 6 months no matter what.